catalog/core/catalog-core-solr-api/src/main/java/ddf/catalog/provider/solr/SolrCollectionProvider.java

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package ddf.catalog.provider.solr;
+
+import com.sun.istack.internal.Nullable;
```

Comment:
❓ Was this maent to be a different Nullable? (the javax.annotations one perhaps)

Reply:
there is no good answer, Nullable has been implement by so many library https://stackoverflow.com/questions/4963300/which-notnull-java-annotation-should-i-use
the javax.annotation seem the most standard but its impl is available in findbugs rather https://stackoverflow.com/questions/19030954/cant-find-nullable-inside-javax-annotation
We currently don't depends on that. But the sun.istack  Nullable is readily available.